frontend/package.json

```diff
@@ -16,13 +16,14 @@
     "babel-plugin-transform-object-assign": "^6.22.0",
     "babel-plugin-transform-proto-to-assign": "^6.26.0",
     "babel-preset-es2015": "^6.24.1",
-    "joi-browser": "^13.0.1"
+    "joi-browser": "^13.0.1",
+    "url-pattern": "^1.0.3"
   },
   "devDependencies": {
     "@shopgate/pwa-core": "latest",
     "@shopgate/pwa-common": "latest",
     "@shopgate/pwa-common-commerce": "latest",
-    "@shopgate/pwa-ui-shared": "^5.6.0-beta.4",
+    "@shopgate/pwa-ui-shared": "latest",
```

Comment:
FYI `latest` will cause issues when the repository is used within the lerna mono repo. I will fix this in   master in all related repos as discussed with waldemar